src/content/platform-integration/ios/c-interop.md

```diff
@@ -63,13 +63,13 @@ API documentation is available from the
 
 [Dart API reference documentation]: {{site.dart.api}}
 
-## Create an FFI plugin
+## Create an FFI package {: #create-an-ffi-plugin }
 
-To create an FFI plugin called "native_add",
+To create an FFI package called "native_add",
```

Comment:
Yeah, not really liking this tendency to replace plugin with package. A plugin is a special type of package and, in my opinion, is more clear than using generic "package".